app/helpers/passengers_helper.rb

```diff
@@ -11,13 +13,30 @@ def doctors_note_row_class(note)
     end
   end
 
-  def passengers_table_row_class(passenger)
-    return if passenger.permanent?
-    return 'no_note' if passenger.doctors_note.blank?
+  def longer_ride(device)
+    if device.try(:needs_longer_rides) == true
+      'longer_ride'
+    end
+  end
 
-    doctors_note_row_class(passenger.doctors_note)
+  def passengers_table_row_class(passenger)
+    if passenger.permanent? && longer_ride(passenger.mobility_device).nil?
```

Comment:
This method seems like it has too many conditionals.

What you're interested in doing is joining two `String`-or-`nil`s together, right?  Actually, you don't even have to join them together. The way this method is used, you could also return an array of strings.